bootstrap_cfn/salt_utils.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+from bootstrap_cfn import utils
+from bootstrap_cfn import errors
+import salt
+import salt.runner
+import salt.client
+import pprint
+import time
+import sys
+
+def start_highstate(target):
+    local = salt.client.LocalClient()
+    jid = local.cmd_async(target, 'state.highstate')
+    return jid
+
+def start_state(target, state):
+    local = salt.client.LocalClient()
+    jid = local.cmd_async(target, 'state.sls', [state])
+    return jid
+
+def state_result(jid):
+    opts = salt.config.master_config('/etc/salt/master')
+    r = salt.runner.RunnerClient(opts)
+    result = r.cmd('jobs.lookup_jid', [jid])
+    if result:
+        return result
+    return False
```

Comment:
This could be writte as "return result if result else False" and get rid lines 25, 26